packages/bun-types/globals.d.ts

```diff
@@ -302,10 +302,20 @@ interface ImportMeta {
    * Does not have a trailing slash
    */
   readonly dir: string;
+  /**
+   * Absolute path to the directory containing the source file. (alias for `dir` to match Node.js)
+   *
+   * Does not have trailing slash
+   */
+  readonly dirname: string;
```

Comment:
i think you opened this before our big `@types/bun` PR. but can you remove these type declarations. they should be provided by `@types/node`